cmd/kube-proxy/app/options_test.go

```diff
@@ -89,94 +89,130 @@ nodePortAddresses:
 `
 
 	testCases := []struct {
-		name               string
-		mode               string
-		bindAddress        string
-		clusterCIDR        string
-		healthzBindAddress string
-		metricsBindAddress string
-		extraConfig        string
+		name                         string
+		mode                         string
+		bindAddress                  string
+		clusterCIDR                  string
+		healthzBindAddress           string
+		metricsBindAddress           string
+		extraConfig                  string
+		expectedHealthzBindAddresses []string
+		expectedHealthzBindPort      int32
+		expectedMetricsBindAddresses []string
+		expectedMetricsBindPort      int32
```

Comment:
Are we likely to end up with additional "expectations" later on in this series of patches? Would it make more sense to just have a function that modifies the expected `kubeproxyconfig.KubeProxyConfiguration`?